docs/0-Preparing/continuing_side_projects.md

```diff
@@ -0,0 +1,53 @@
+## Continuing side projects
+
+This section continues the discussion on side projects. It's meant to go in a little more depth if you already have some coding and side project experience, but it might still be an interesting read otherwise.
+
+### More signals that side projects can give the employer
+
+**You can work in a team**
+
+- *Example*: You make a platformer game with 2 friends and publish it online. It only has one hour of gameplay but it works great!
+- *Why it's useful*: You'll have to work in a team at some point
+
+**You can go in depth in a subject**
+
+- *Example*: You really get into computer vision and read a bunch of papers about recognizing different types of architecture from pictures
+- *Why it's useful*: A more advanced version of learning a concept on your own
```

Comment:
In this case since I'm talking about signaling, I'm talking about skills useful to the employer (or equivalently, useful to the student because it's useful to the employer).